README.md

```diff
@@ -183,6 +183,110 @@ Blank nodes are not supported.
   }
 }
 ```
+#### Resource Object Fragmentation Shape
+
+A fragmentation strategy that group’s triples by (subject) resources.
+It generates shapes and shape index files in the root of the iri container for each resource type defined in the config. 
+The `shape` must be a a list of path to shape template following the [`ShExC`](https://shex.io/shex-semantics/index.html#shexc) format
+(the templating parameters are define below),
```

Comment:
Could you follow the structure for describing options as here: https://github.com/SolidBench/rdf-dataset-fragmenter.js/?tab=readme-ov-file#file-quad-sink